scripts/build-upload-a-docker-image.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -exu
+
+base_debug_img_arg=""
+docker_file_arg="Dockerfile"
+target_arg=""
+local_test_only='N'
+platforms="linux/amd64"
+name_space="jaegertracing"
+
+while getopts "lbc:d:f:p:t:" opt; do
+	case "${opt}" in
+	c)
+		component_name=${OPTARG}
+		;;
+	b)
+		base_debug_img_arg="--build-arg base_image=localhost:5000/baseimg_alpine:latest --build-arg debug_image=localhost:5000/debugimg_alpine:latest "
+		;;
+	d)
+		dir_arg=${OPTARG}
+		;;
+    f)
```

Comment:
The indentation seems inconsistent between the flags.

Reply:
issue solved. Thanks!